pkg/gadgets/trace/bind/types/types.go

```diff
@@ -15,20 +15,25 @@
 package types
 
 import (
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	Pid       uint32 `json:"pid,omitempty"`
-	Comm      string `json:"comm,omitempty"`
-	Protocol  string `json:"proto,omitempty"`
-	Addr      string `json:"addr,omitempty"`
-	Port      uint16 `json:"port,omitempty"`
-	Options   string `json:"opts,omitempty"`
-	Interface string `json:"if,omitempty"`
-	MountNsID uint64 `json:"mountnsid,omitempty"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,width:7"`
+	Comm      string `json:"comm,omitempty" column:"comm,width:16"`
+	Protocol  string `json:"proto,omitempty" column:"proto,width:6"`
+	Addr      string `json:"addr,omitempty" column:"addr,width:16"`
+	Port      uint16 `json:"port,omitempty" column:"port,width:7"`
+	Options   string `json:"opts,omitempty" column:"opts,width:7"`
+	Interface string `json:"if,omitempty" column:"if,width:7"`
```

Comment:
This is the name of a network interface. I think 7 is a bit short. What about 12?

Reply:
Aahh, I thought it was the interface index because, [by default, it is `0`](https://github.com/kinvolk/inspektor-gadget/blob/0d7c7f0d0403cee0b2394f671940f606878fc687/pkg/gadgets/trace/bind/tracer/tracer.go#L275). I think we should change that default, it should be an empty string, don't you think?

Beyond that, you are right. I will change it to 12.

Reply:
Yes, this was on my TODO list -> https://github.com/kinvolk/inspektor-gadget/pull/978.